SteamList/Views/GameTableViewCell.swift

```diff
@@ -12,6 +12,7 @@ class GameTableViewCell: UITableViewCell {
     
     override func awakeFromNib() {
         super.awakeFromNib()
+        selectionStyle = .none
```

Comment:
You can safely move selection style to xib file since it is already responsible for layout

Reply:
done